packages/wdio-cli/src/launcher.js

```diff
@@ -122,15 +112,31 @@ class Launcher {
          * schedule test runs
          */
         let cid = 0
-        for (let capabilities of caps) {
+        if (this.isMultiremote) {
+            /**
+             * Multiremote mode
+             */
             this.schedule.push({
                 cid: cid++,
-                caps: capabilities,
-                specs: this.configParser.getSpecs(capabilities.specs, capabilities.exclude).map(s => ({ files: [s], retries: config.specFileRetries })),
-                availableInstances: capabilities.maxInstances || config.maxInstancesPerCapability,
-                runningInstances: 0,
-                seleniumServer: { hostname: config.hostname, port: config.port, protocol: config.protocol }
+                caps,
+                specs: this.configParser.getSpecs(caps.specs, caps.exclude).map(s => ({ files: [s], retries: config.specFileRetries })),
+                availableInstances: config.maxInstances || 1,
```

Comment:
Why 1 instead of maxInstancesPerCapability?

Reply:
Before it was allowed to have only 1 instance in multiremote, even maxInstances was ignored.
I've noticed that maxInstances value is 100 and was afraid it might brake something in Multiremote, so I decided to use 1 instead of maxInstancesPerCapability

I can change it to something like `maxInstancesPerCapability / (Object.keys(caps).length || 1)` to avoid spawning to much browser instances in Multiremote. Not sure if it makes sense to do that.

Reply:
I can only find a few references to maxInstancesPerCapability, this actually seems deprecated: https://webdriver.io/docs/options.html

Reply:
ah sorry, one more comment, it makes no sense to use `maxInstancesPerCapability`, we should only use `maxInstances`, so that's fine.
Sorry for confusing, `maxInstancesPerCapability` only makes sense in regular mode